.github/pull_request_template.md

```diff
@@ -1,14 +1,14 @@
-Resolves: *(direct link to the issue)*
+Resolves: #NNNNN <!-- Replace `NNNNN` with a GitHub issue number, or a direct link if the issue is not on GitHub -->
 
-*(short description of the changes and the motivation to make the changes)*
+<!-- Add a short description of and motivation for the changes here -->
 
-<!-- Use "x" to fill the checkboxes below like [x] -->
+<!-- Replace `[ ]` with `[x]` to fill the checkboxes below -->
 
-- [ ] I signed [CLA](https://musescore.org/en/cla)
-- [ ] I made sure the code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
-- [ ] I made sure the code compiles on my machine
-- [ ] I made sure there are no unnecessary changes in the code
-- [ ] I made sure the title of the PR reflects the core meaning of the issue you are solving
-- [ ] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
-- [ ] I made sure the commit message title starts with "fix #424242:" if there is a related issue
-- [ ] I created the test (mtest, vtest, script test) to verify the changes I made
+- [ ] I signed the [CLA](https://musescore.org/en/cla)
+- [ ] The title of the PR describes the issue it addresses
+- [ ] Each commit's message describes its purpose and effects and references the issue it resolves
```

Comment:
I'd personally drop the 's from commit here; but I'm non-native, so what do I know

Reply:
Either would work, though keeping `'s` is slightly more correct because it makes it clear we are talking about the purpose of the commit, not the purpose of the commit *message*.

However, we need a comma after "effects". It should be:

> Each commit's message describes its purpose and effects, and references any issues it resolves.

And I suggest rewording the one above to be:

> The title of the PR describes the ~~issue~~ __problem__ it addresses.

This avoids using the word "issue" to mean a generic problem as well as a specific numbered issue from our tracker.